consensus/src/consensus.rs

```diff
@@ -115,7 +112,7 @@ impl<T: Operations + 'static, D: Database + 'static> Consensus<T, D> {
         let sender =
             AgreementSender::new(self.agreement_process.inbound_queue.clone());
 
-        // Consensus loop - generation-selection-reduction loop
+        // Consensus loop - generation-proposal-reduction loop
```

Comment:
This should be `proposal-validation-ratification`

Reply:
Done